### ISSUE TITLE
Get rid of +/- buttons and enable directly typing in the quantity (e.g. in the Logistics Request Pipe)

### ISSUE THREAD

**Issue by mikeloeven** [RS485/LogisticsPipes#827]
With how far minecraft has advanced i am shocked that were still using the archaic plus and minus button system for request quantities. while this method mostly works when your dealing with large requests or anything that requires precice fluid values such as tinkers construct casting with fluid pipes the plus and minus buttons are simply a major pain in the ass and there is no need for them at all there never was and it was always a horrible UI design. Please for the love of god get rid of the buttons and just put in a text field so we can enter the quantity we want in literally a fraction of the time it takes to fuddle with the buttons

The issue with just how badly the +/- buttons perform becomes glaringly obvious when you take into account fluid request pipes 
- up 1 
  ++ up a bucket 1000
  +++ up 16 buckets 

when using this with tinkers construct to pull molten metal into a casting table your dealing with variable amounts of liquid metals from 1 - 500mb so literally the scale of the buttons is simply not appropriate and without a text field which you can enter an integer manually you end up wasting a lot of time and developing carpel tunnel syndrome making what should be a simple and east task tedious and not worth putting up with

And I am sorry if there is already some keyboard shortcut to perform this operation however the documentation is pretty spotty in terms of the "hidden" keystrokes

**Comment by davboecki** [comment]
Just for the time beeing you can use the mouse wheel in compination with the shift and the ctrl key in different combinations to change the page and the number faster than the buttons.

**Comment by Morketh** [comment]
Could you also include this UI feature in the Disk aswell? I decided to create a disk as a work around for the materials list for the Buildcraft builder some of my builds were in excess of 8 Thousand StoneBrick took me a few hours to set of 3 disks for the request pipes

**Comment by loordgek** [comment]
@Morketh for that there is https://github.com/RS485/LogisticsPipes/issues/435

**Comment by Morketh** [comment]
@loordgek yes that would defiantly fix the builder (already made a post there awhile ago) I was merely giving an Example of why having a text box on the Disk would be a good thing. as another example maybe you have a void-chest/pipe and you would like to dump 8 thousand cobblestone at a time after you realized your stock grew to 100K you could have pre programmed disks that you could plop into the pipe no mess, no fuss. or maybe you have a store set up with a trading mod that youd like to walk up to drop a disk in the pipe and collect your items. you could keep "shopping lists" of any item you wanted for any project in a single item card that you could simplly slap into a pipe with out having to count every item and request them one by one or item-stack by item-stack.
